ui/pages/confirmation/confirmation.js

```diff
@@ -185,23 +183,20 @@ export default function ConfirmationPage() {
         </div>
       )}
       <div className="confirmation-page__content">
-        <Box justifyContent="center">
-          <NetworkDisplay
-            colored={false}
-            indicatorSize={SIZES.XS}
-            labelProps={{ color: COLORS.BLACK }}
-          />
-        </Box>
-        <Box justifyContent="center" padding={[1, 4, 4]}>
-          <Chip
-            label={stripHttpsScheme(originMetadata.origin)}
-            leftIcon={
-              <SiteIcon
-                icon={originMetadata.iconUrl}
-                name={originMetadata.hostname}
-                size={32}
-              />
-            }
+        {templatedValues.networkDisplay ? (
+          <Box justifyContent="center">
+            <NetworkDisplay
+              colored={false}
+              indicatorSize={SIZES.XS}
+              labelProps={{ color: COLORS.BLACK }}
+            />
+          </Box>
+        ) : null}
+        <Box justifyContent="center" padding={[4, 4, 4]}>
+          <SiteOrigin
```

Comment:
We could split out this change to how the site origin is shown on templated pages

Reply:
@hmalik88

Reply:
This is from my MetaMask account